src/Controller/Exception.php

```diff
@@ -245,11 +246,21 @@ protected function getContextArray(\Exception $exception = null)
             $snippet = false;
         }
 
+        // We might or might not have $this->app['request'] yet, which is used in the
+        // template to show the request variables. Use it, or grab what we can get.
+        if ($this->app['request'] instanceof Request) {
+            $request = $this->app['request'];
```

Comment:
If one doesn't, the other doesn't. The sole difference being that the request gets added to the stack during the request cycle…

The request stack is added in the `Application` [constructor](https://github.com/silexphp/Silex/blob/1.3/src/Silex/Application.php#L130) so if we don't have it, we've crashed in bootstrap and … all … hope … is … lost! :laughing: